File: src/com/joelbyler/mastermind/ruboto/MasterMindLaunchActivity.java
package com.joelbyler.mastermind.ruboto;

import android.os.Bundle;

public class MasterMindLaunchActivity extends org.ruboto.EntryPointActivity {
	public void onCreate(Bundle bundle) {
		getScriptInfo().setRubyClassName(getClass().getSimpleName());
	    super.onCreate(bundle);
	}
}

File: src/master_mind_launch_activity.rb
require 'ruboto/widget'

# Wrapper class, needed in order to have menu on main screen
class MasterMindLaunchActivity

  def on_create(bundle)
    super
    i = android.content.Intent.new
    i.setClassName($package_name, "#{$package_name}.MasterMindMainActivity")
    startActivity(i)
    finish
  end
end
